client/material/left-nav/left-nav.jsx

```diff
@@ -4,7 +4,11 @@ import styled from 'styled-components'
 
 import { colorBackground } from '../../styles/colors'
 
-const Footer = styled.div``
+const Footer = styled.div`
+  position: sticky;
```

Comment:
I don't think any of these changes are are actually necessary, I think the overflow-y accomplishes everything we need (since the `Footer` is not contained within the `Sections`)